src/libraries/System.Net.Sockets/tests/FunctionalTests/CreateSocketTests.cs

```diff
@@ -445,6 +455,194 @@ public async Task Ctor_SafeHandle_Listening_Success(bool shareSafeHandle)
             }
         }
 
+        [DllImport("libc")]
+        private static extern int socket(int domain, int type, int protocol);
+
+        private const int PF_NETLINK = 16;
+
+        private class NlEndPoint : EndPoint
+        {
+            [StructLayout(LayoutKind.Sequential)]
+            internal struct sockaddr_nl
+            {
+                internal ushort sin_family;
+                private ushort pad;
+                internal int pid;
+                private int nl_groups;
+            }
+
+            private readonly int _pid;
+
+            public NlEndPoint(int pid)
+            {
+                _pid = pid;
+            }
+
+            public override AddressFamily AddressFamily
+            {
+                get
+                {
+                    return AddressFamily.Unknown;
```

Comment:
This isn't one of the ones we added an address family for?

Reply:
correct . We added Netlink and then took back as it was not ready.